compiler/extccomp.nim

```diff
@@ -83,6 +83,31 @@ compiler gcc:
     props: {hasSwitchRange, hasComputedGoto, hasCpp, hasGcGuard, hasGnuAsm,
             hasAttribute})
 
+# GNU C and C++ Compiler
+compiler aarch64NoneElfGCC:
```

Comment:
Nitpick but I would name this `switchGCC`. We might have a generic ``aarch64-none-elf-gcc`` in the future (I guess?)

Reply:
:+1: